src/latticpp/ckks/bootstrap.cpp

```diff
@@ -7,11 +7,14 @@ using namespace std;
 
```

Comment:
We have an issue in all CPP files with the formatting. We need to keep the same formatting as before